Product.cpp

```diff
@@ -0,0 +1,129 @@
+#include "Product.h"
+#include <iostream>
+#include <cstring>
+#include <fstream> 
+
+using namespace std;
+
+// Use static global file stream for product file operations
+static std::fstream productFile;
+
+
+// Default constructor
+Product::Product() { }
+
+// Initialize the product file
+void Product::initProduct() {
+    // open the binary file 
+    productFile.open("product.bin", std::ios::binary);
```

Comment:
russ used .bat in his slides. I think maybe we should use that

Reply:
ok will fix